kotlin/export-tiles/README.md

```diff
@@ -0,0 +1,37 @@
+# Export tiles
+
+Download tiles to a local tile cache file stored on the device.
+
+![Export Tiles Sample](export-tiles.png)
```

Comment:
Was there a settled way to do this during the great README migration?

![Image of Export Tiles sample] or something like that?

Reply:
ah yes you're right, thanks for spotting that: should be just "image of export tiles", I've corrected it